lib/Alchemy/Phrasea/Controller/Root/AccountController.php

```diff
@@ -299,13 +306,102 @@ public function displayAccount()
         $manager = $this->getEventManager();
         $user = $this->getAuthenticatedUser();
 
+        $repo_baskets = $this->getBasketRepository();
+        $baskets = $repo_baskets->findActiveValidationAndBasketByUser($user);
+
+        $apiAccounts = $this->getApiAccountRepository()->findByUser($user);
+
+        $ownerPublisher = $this->getFeedPublisherRepository()->findBy(['user' => $user, 'owner' => true]);
```

Comment:
semantic : rename as $ownedFeeds to match the type of objects found in repo.

Reply:
done